scripts/serverSetup.md

```diff
@@ -0,0 +1,50 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 is installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME. Additionally, this system uses either Python 3.5 or 3.6.**
+
+## Deploying the web sever
+In the yw_website directory, change directory into the scripts directory. You should see a number of bash scripts:
+```
+deploy_python.sh
+deploy_python3.sh
+workflow_sample.gv
```

Comment:
This is still in the Readme but no longer in the repo

Reply:
fixed